src/listener.ts

```diff
@@ -0,0 +1,21 @@
+import { NestFactory } from "@nestjs/core";
+import { AppModule } from "./app.module";
+import { MicroserviceOptions, Transport } from '@nestjs/microservices';
+
+async function bootstrap() {
+    const app = await NestFactory.createMicroservice<MicroserviceOptions>(AppModule, {
+        transport: Transport.RMQ,
+        options: {
+            urls: [process.env.RABBITMQ_QUEUE],
+            queue: process.env.RABBITMQ_URL,
+            queueOptions: {
```

Comment:
Why are you creating this file? Your task is to connect identity srvice to notification service.

In this case the I-S sends messages to N-S, so identity cannot have a listener file, Nevertheless... listener has been created in the N-S so you don't have any business with listener.ts

Reply:
Does connecting the identity service to the notification service not mean listening to messages from the notification service?

The last part was a mistake